doc/appendices/administration.rst

```diff
@@ -0,0 +1,120 @@
+   technical implementation of the codebase to meet the long-term project
+   goals. This should be the same person as the "core capability lead"
+   described in the PMP. One key responsibility of the code lead is to set up
+   milestones for releases and execute the release process described below.
+
+Maintainer
+   Maintainers should be familiar with most or all parts of the codebase and
+   are responsible for merging pull requests and closing issues.
+
+Core team member
+   The "core team" are those currently funded to work on Celeritas. Core team
+   members are responsible for reviewing pull requests in accordance with the
+   :ref:`contributing guidelines <contributing>`. They should regularly
+   contribute new code or publish new results using Celeritas.
+
+Contributor
+   Anyone can submit a pull request that conforms to the contribution
+   guidelines! Contributors are recognized by including their handle and pull
+   requests in release notes.
+
+.. _@sethrj: https://github.com/sethrj
+
+
+Release process
+===============
+
+Celeritas uses `Semantic Versioning`_ to enumerate releases. During its initial
+development phase, ``0.x.0`` is a major release and ``0.x.z`` is a patch
+release. When Celeritas is declared stable, ``x.0.0`` is a major release,
+``x.y.0`` is a minor release, and ``x.y.z`` is a patch release.
+
+Major and minor releases (including 0.x.0 development releases) must have a
+milestone in the git issue tracker with a list of issues that can be assigned.
+Only major releases can deprecate or remove features and change
+:ref:`public-facing APIs <api>`. Both major and minor releases should include
+notable improvements to the code.
+
+Patch releases can be created at any time but should typically include at least
+one critical bug fix or several substantial fixes. Patch releases should focus
+almost exclusively on fixes and should generally not include new features or
+other major code changes.
+
+.. _Semantic Versioning: https://semver.org
+
+
+Releases can be created from the master branch (major, minor, patch) or a
+"backport" branch (minor, patch). The following process must be followed (and
+may need iteration to converge) for each release.
+
+- Ensure all CI jobs passed for the release in question. This is automatic for
+  releases from master (since every pull request must pass) but should be
+  checked manually for backports.
+- Run regression tests on Summit, Crusher, Wildstyle to check for major
+  performance regressions or newly failing tests. Postpone the release
+  temporarily if major new bugs are detected. If minor updates are needed to
+  fix the build or tests on a particular machine, include those as part of the
+  "pre-release" pull request that includes new documentation.
+- [TODO: define high-level validation tests and a test matrix correlating
+  capability areas (code files/directories changed) to test names.] Rerun
+  and check all validation tests that might be affected by changes since the
```

Comment:
> that might be affected by changes 

Why limit the test to an arbitrary list?  What is the downside to run "all" the tests for each releases?  I.e. the pitfall is if the "guess" of which test "might" be affected is wrong, the release might be broken in a way that could have been noticed.

Reply:
This bullet was stimulated by some of the presentations at the geant4 meeting. It specifically is targeted at validation tests that don't have automated statistical checks but rather need an analyst to validate the results. There's going to be a balance between the person-power cost of extra (possibly unnecessarily) validation to reduce the number of bugs introduced, and the agility of new releases.

Reply:
I have to assume that it is/shoud be more along the line of "failure to match previous results might not be an error" and consequently they could still but run automatically but failure would not be fatal.  (And one could argue that if a test is not supposed to be altered by a change and it is then it is almost as interesting as the other cases [exclusion for "unstable" test].   In addition to the "person-power", there is also the fact that there can be a very large set of such test and it can become computationally expensive.

Either way, you may to specify more what is implied by "validation tests".